tokio/src/task/blocking.rs

```diff
@@ -39,10 +39,11 @@ cfg_rt_threaded! {
 cfg_blocking! {
     /// Runs the provided closure on a thread where blocking is acceptable.
     ///
-    /// In general, issuing a blocking call or performing a lot of compute in a future without
-    /// yielding is not okay, as it may prevent the executor from driving other futures forward.
-    /// A closure that is run through this method will instead be run on a dedicated thread pool for
-    /// such blocking tasks without holding up the main futures executor.
+    /// Runs the provided function on the executor reserved for blocking operations.
```

Comment:
Can we fix the duplicated line?

Reply:
I have fixed it.

Reply:
It is not running on a separate executor, but a thread on the same executor. I would rather swap them the other way.

Reply:
I've just realized that it was I who messed up my clipboard previously. When I said "It can be thought of as an efficient Iterator for collections of bytes.", I was trying to say "Runs the provided closure on a thread where blocking is acceptable."

Reply:
Okay, I will commit the final change